requirements/dev.txt

```diff
@@ -275,6 +275,8 @@ repoze-sendmail==4.3
     # via
     #   -r requirements/requirements.txt
     #   pyramid-mailer
+rfc3339-validator==0.1.4
```

Comment:
I just magically springs to life and provides the "format=date-time" used in the schema. If the module is installed then `jsonschema` will validate this format, otherwise it ignores it and will accept anything.